src/updater/updater.cpp

```diff
@@ -120,10 +120,16 @@ auto mmapFile(const std::string &filename, boost::interprocess::mode_t mode)
     using boost::interprocess::file_mapping;
     using boost::interprocess::mapped_region;
 
+    {
+        storage::io::FileReader file(filename, storage::io::FileReader::VerifyFingerprint);
```

Comment:
Yes, it should be close-on-destroy.  The errors on appveyor seem to be around `mmap` calls though - perhaps we're not unmapping at some point?  Multiple mappings should work though.....